rlp/src/stream.rs

```diff
@@ -344,6 +344,13 @@ impl RlpStream {
 		self.note_appended(1);
 		self.finished_list = true;
 	}
+
+	/// Finalize current unbounded list. Panics if no unbounded list has been opened.
+	#[deprecated(since = "0.4.3", note = "use finalize_unbounded_list instead")]
+	pub fn complete_unbounded_list(&mut self) {
+		self.finalize_unbounded_list();
```

Comment:
I'd expect the compiler to inline this call so there'd not be a performance hit here. Would be fantastic to have a proof of that (and if I'm wrong add the `#[inline]` annotation to `finalize_unbounded_list()`).